### DISCUSSION TITLE
kruise-daemon部署咨询

### DISCUSSION THREAD

**Discussion by steven6718** [openkruise/kruise#1055] [Q&A]
最近在使用openKruise中，遇到部署kruise-daemon时，因为template中，使用了hostNetwork: true，导致某些node上端口冲突。看chart包中，kruise-manager的hostNetwork选项是暴露给用户可修改的，但是kruise-daemon没有暴露，想咨询下修改kruise-daemon的hostNetwork配置为false，使用容器网络会有什么问题？基于什么原因没有把kruise-daemon的hostNetwork选项暴露给用户选择，谢谢~

**Comment by FillZpp** [comment]
目前版本 kruise-daemon 配置 hostNetwork=false 应该没什么问题，但后续版本中可能还会有功能需要调用本节点上 kubelet 的端口，所以尽量还是使用 hostNetwork 模式。如果端口冲突，建议通过配置 daemon.port 来修改端口。
